guide/airline_reservation/airline_reservation_service.bal

```diff
@@ -0,0 +1,268 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+import ballerina/mysql;
+import ballerina/observe;
+import ballerina/runtime;
+import ballerina/sql;
+//import ballerinax/docker;
+//import ballerinax/kubernetes;
+
+//@docker:Config {
+//    registry:"ballerina.guides.io",
+//    name:"airline_reservation_service",
+//    tag:"v1.0"
+//}
+//
+//@docker:Expose{}
+
+//@kubernetes:Ingress {
+//  hostname:"ballerina.guides.io",
+//  name:"ballerina-guides-airline-reservation-service",
+//  path:"/"
+//}
+//
+//@kubernetes:Service {
+//  serviceType:"NodePort",
+//  name:"ballerina-guides-airline-reservation-service"
+//}
+//
+//@kubernetes:Deployment {
+//  image:"ballerina.guides.io/airline_reservation_service:v1.0",
+//  name:"ballerina-guides-airline-reservation-service"
+//}
+
+// Service endpoint
+endpoint http:Listener airlineEP {
+    port:9091
+};
+
+// Airline reservation service
+@http:ServiceConfig {basePath:"/airline"}
+service<http:Service> airlineReservationService bind airlineEP {
+
+    // Resource 'flightQatar', which checks about airline 'Qatar Airways'
+    @http:ResourceConfig {
+        methods:["POST"],
+        path:"/qatarAirways",
+        consumes:["application/json"],
+        produces:["application/json"]
+    }
+    flightQatar (endpoint caller, http:Request request) {
+        http:Response response;
+        json reqPayload;
+
+        string resourcePath = "/airline/qatarAirways";
+        log:printDebug("Received at : " + resourcePath);
+        // Try parsing the JSON payload from the request
+        match request.getJsonPayload() {
+            // Valid JSON payload
+            json payload => reqPayload = payload;
+            // NOT a valid JSON payload
+            any => {
+                response.statusCode = 400;
+                response.setJsonPayload({"Message":"Invalid payload - Not a valid JSON payload"});
+                _ = caller->respond(response);
```

Comment:
shall we handle the possible errors, maybe with `but`, wherever applicable instead of ignoring them? 

https://github.com/ballerina-platform/ballerina-examples/blob/master/examples/hello-world-service/hello_world_service.bal#L14